chef/cookbooks/hyperv/templates/default/nova.conf.erb

```diff
@@ -31,13 +28,21 @@ rabbit_virtual_host=<%= @rabbit_settings[:vhost] %>
 logdir=<%= @openstack_log%>\
 logfile=nova-compute.log
 network_api_class=nova.network.neutronv2.api.API
-neutron_url=<%= @neutron_protocol%>://<%= @neutron_server_host%>:<%= @neutron_server_port%>
-neutron_auth_strategy=keystone
-neutron_admin_tenant_name=<%= @keystone_settings['service_tenant'] %>
-neutron_admin_username=<%= @neutron_service_user%>
-neutron_admin_password=<%= @neutron_service_password%>
-neutron_admin_auth_url=<%= @keystone_settings['internal_auth_url'] %>
-neutron_api_insecure=<%= @neutron_insecure ? 'True' : 'False' %>
+
+[neutron]
+url=<%= @neutron_protocol%>://<%= @neutron_server_host%>:<%= @neutron_server_port%>
+auth_strategy=keystone
+admin_tenant_name=<%= @keystone_settings['service_tenant'] %>
+admin_username=<%= @neutron_service_user%>
+admin_password=<%= @neutron_service_password%>
+admin_auth_url=<%= @keystone_settings['internal_auth_url'] %>
+api_insecure=<%= @neutron_insecure ? 'True' : 'False' %>
```

Comment:
Can you add something like: `region_name = <%= @keystone_settings['endpoint_region'] %>` ?